packages/util/src/index.ts

```diff
@@ -1,12 +1,7 @@
 export * from "./addTrailingSlash"
 export * from "./BigMath"
-export * from "./blake2Concat"
 export * from "./classNames"
-export * from "./convertAddress"
-export * from "./decodeAnyAddress"
-export * from "./decodeSs58Format"
 export * from "./deferred"
-export * from "./encodeAnyAddress"
 export * from "./firstThenDebounce"
```

Comment:
encodeAnyAddress/decodeAnyAddress is still used in Portal, so we'll need to remember to switch it to use the new one when we upgrade to the latest version of `@talismn/util`.